### ISSUE TITLE
Redirect Response error

### ISSUE THREAD

**Issue by acurrieclark** [octoper/statamic-html-minify#2]
### Description

If a redirect response is returned from a route, an exception is thrown:

```
Return value of Octoper\HtmlMinify\HtmlMinifyMiddleware::handle() must be an instance of Illuminate\Http\Response, instance of Illuminate\Http\RedirectResponse returned
```

Might a return type of `Symfony\Component\HttpFoundation\Response` work? I think that all `Illuminate` responses inherit from it.

**Comment by octoper** [comment]
Fixed with this pull request #3

**Comment by acurrieclark** [comment]
That's great. Thanks for the quick turnaround.
